fv3core/stencils/fillz.py

```diff
@@ -99,49 +97,47 @@ def fix_tracer(
             q = fac * dm / dp if fac * dm / dp > 0.0 else 0.0
 
 
-def compute(
-    dp2: FloatField,
-    tracers: Dict[str, Any],
-    im: int,
-    jm: int,
-    km: int,
-    nq: int,
-):
-    # Same as above, but with multiple tracer fields
-    i1 = spec.grid.is_
-    j1 = spec.grid.js
+class Fillz:
```

Comment:
Proposal for class name inspired by the name-list doc: `FillNegativeTracerValues`

Reply:
Implemented, thank you!